sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn

```diff
@@ -1252,7 +1252,7 @@ or (calculatedDataPartnerFedralProvincialPDReceiptCost &gt; 0)</bpmn:conditionEx
           <zeebe:output source="=if studentDataGovernmentFundingCosts = null&#10;then 0&#10;else&#10;  studentDataGovernmentFundingCosts" target="studentDataGovernmentFundingCosts" />
           <zeebe:output source="=if studentDataNongovernmentFundingCosts = null&#10;then 0&#10;else&#10;  studentDataNongovernmentFundingCosts" target="studentDataNongovernmentFundingCosts" />
           <zeebe:output source="=if studentDataParentVoluntaryContributionsCosts = null&#10;then 0&#10;else&#10;  studentDataParentVoluntaryContributionsCosts" target="studentDataParentVoluntaryContributionsCosts" />
-          <zeebe:output source="=if studentPDStatus = null&#10;then false&#10;else&#10;  studentPDStatus" target="studentPDStatus" />
+          <zeebe:output source="=if studentDataApplicationPDPPDStatus = &#34;yes&#34;&#10;then true&#10;else&#10;  false" target="calculatedDataApplicationPDPPDStatus" />
```

Comment:
Just as a suggestion, `calculatedDataPDPPDStatus` would be enough IMO because at this stage the application scope does not matter and in the future, it may include Student PD/PPD when we need a reassessment in case we do not get the answer from ATBC, which means that the "application scope" will be overwritten.